devbox.go

```diff
@@ -313,17 +313,50 @@ func missingDevboxJSONError(dir string) error {
 			}
 		}
 	}
-	return usererr.New("No devbox.json found in %s, or any parent directories. Did you run `devbox init` yet?", dir)
+
+	parentDirCheckAddendum := ""
+	if didCheckParents {
+		parentDirCheckAddendum = ", or any parent directories"
+	}
+
+	return usererr.New("No devbox.json found in %s%s. Did you run `devbox init` yet?", dir, parentDirCheckAddendum)
 }
 
+// findConfigDir
 func findConfigDir(dir string) (string, error) {
```

Comment:
Should this go in `config.go`?

Reply:
its added inside devbox.go to keep the CLI minimal. Moving this logic inside here enables `devbox` as a package to encapsulate all the logic and functionality (enabling reuse outside this CLI scenario).

Reply:
I agree with keeping it in `devbox`. I was wondering if it should go in `config.go` within the same package.